### PR TITLE
Travis/level current badge

### PR DESCRIPTION
**Pull request by traversng**
Add current badge to list cards for levels page
Debounce treasure box calls
Optimize treasure box reveal steps state (local storage)
Move click even up from ellipsis on levels card to the entire card
Add cursor hovers accordingly
New pill component
Quick fix for superflous api calls to treasurebox endpoint

### CODE REVIEW

File: src/app/(drawerPages)/art-reveal/page.tsx [hunk 1]
```diff
@@ -72,6 +63,31 @@ export default function ArtReveal() {
...
+        user: {
+          address: address!,
+          cbId: userPublicProfile?.subdomainProfile?.name,
+          ensName: userPublicProfile?.ensDomainProfile?.name,
+        },
+      });
+    }, 500),
+    [
+      address,
+      // attackBox,
```
Comment (siwan-cb) [inline src/app/(drawerPages)/art-reveal/page.tsx:79]: did this caused infinite rendering?

File: src/hooks/useTreasureBox.ts [hunk 2]
```diff
@@ -37,6 +37,7 @@ export function useTreasureBox({ gameId }: Props) {
         console.error(error);
         // Handle error appropriately
       },
+      staleTime: 1000 * 30, // 30 seconds
```
Comment (siwan-cb) [inline src/hooks/useTreasureBox.ts:40]: 👍

File: src/app/(drawerPages)/art-reveal/page.tsx (hunk 1 above)
Comment (traversng) [reply src/app/(drawerPages)/art-reveal/page.tsx:79]: Actually no, but we weren't handling spam clicks

**Comment by siwan-cb** [comment]
Level card isn't navigating when pressed but text and svgs are working as expected. Could be from merge conflict. I can try to fix it tonight
not-working

**Pull request merged**